bentoml/_internal/utils/trie.py

```diff
@@ -0,0 +1,46 @@
+import typing as t
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    TrieDataType: t.TypeAlias = t.Union[str, t.Dict[str, "Trie"]]
```

Comment:
```suggestion
    TrieDataType: "t.TypeAlias" = "t.Union[str, t.Dict[str, Trie]]"
```

Reply:
I'd tried this but because `TypeAlias` is special you can't quote it. Just found out you *can* quote the alias itself, though, so I'll do that.